trustpinning/trustpin.go

```diff
@@ -37,6 +37,8 @@ func NewTrustPinChecker(trustPinConfig TrustPinConfig, gun string) (CertChecker,
 	}
```

Comment:
Sorry, I can't comment on the exact line, but maybe in the block above on line 35 we might want to add a `logrus.Debugf("Trust-pinning using Cert IDs")` - possibly with the cert ID's listed or just with the len of the cert IDs?